### ISSUE TITLE
Should ExpressionBodySyntaxLineBreaks take into account maxLineLength?

### ISSUE THREAD

**Issue by MyDogTom** [arturbosch/detekt#130]
I have the following code
```kotlin
fun <T> ObservableSource<T>.toV1Observable(strategy: BackpressureStrategy): Observable<T> =
      RxJavaInterop.toV1Observable(this, strategy)
```
`ExpressionBodySyntaxLineBreaks` fails here. But I cannot make it one line, because it will be longer than 120 characters.
This is top level function. `@Suppress` for only one function doesn't work :(

**Comment by MyDogTom** [comment]
Same for `ExpressionBodySyntax`

**Comment by arturbosch** [comment]
`ExpressionBodySyntaxLineBreaks` wants you to have the `=` on the same line as the expression, so 
` = RxJavaInterop.toV1Observable(this, strategy)` should work too.

This both rules also do not consider indentation, which is not good either. But I do not want to spend too much of my time on formatting things. We can now use a local intellij to format code with the `detektIdeaFormat` task and there is ktlint where I copy fixes for the existing rules. If ktLint users get more active and implement more rules, I will try to integrate ktlint as a rule set.

So basically run `detektFormat` and then `detektIdeaFormat` will do the job :)
